docs/news.rst

```diff
@@ -1262,6 +1262,10 @@ Deprecations
 -   :func:`scrapy.utils.response.response_httprepr` is now deprecated.
     (:issue:`4972`)
 
+-   :func:`scrapy.core.downloader.Downloader._get_slot_key` is now deprecated.
+    Consider using its corresponding public method get_slot_key() instead.
+    (:issue:`6340`)
+
```

Comment:
This is a section for an old release. To cover this new change, you should either add it at the very beginning of the file, in a new section for an unreleased version, or do nothing, and hope we don’t forget to cover this when the build the release notes. The first approach is better, but the second approach is the usual and fine as well.

Reply:
Ahh ! I see, thank you for pointing this out